geocatbridge/servers/models/geoserver.py

```diff
@@ -1062,13 +1063,18 @@ def clearWorkspace(self, recreate=True) -> bool:
                 url = f"{self.apiUrl}/workspaces/{self.workspace}/styles/{style['name']}.json?recurse=true&purge=true"
                 self.request(url, method="delete")
 
+            # Get isolation flag
+            url = f"{self.apiUrl}/workspaces/{self.workspace}.json"
+            workspace = self.request(url).json()
+            isolated = workspace["workspace"]["isolated"]
```

Comment:
Hi @kannes, I prefer a safer way of getting that property. What I usually do is something like:

```suggestion
            isolated = workspace.get("workspace", {}).get("isolated", False)
```

This way, you will never get a `KeyError` and just use a sensible default if a property is missing (in my experience, you can never trust the GeoServer REST API 😉).

Reply:
Oh yes, I share that :)